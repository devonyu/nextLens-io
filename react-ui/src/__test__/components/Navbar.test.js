import React from 'react';
import Navbar from '../../components/NavBar';
import renderer from 'react-test-renderer';

test('Can see navbar', () => {
  const userState = {
    about: '',
    email: '',
    firstname: '',
    id: 0,
    mount: 1,
    profileimgurl: ''
  };
  const emptyFunc = () => {};
  const component = renderer.create(
    <Navbar
      userInformation={userState}
      loggedIn={false}
      sidebar={emptyFunc}
      changeState={emptyFunc}
      changeView={emptyFunc}
    />
  );
  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
