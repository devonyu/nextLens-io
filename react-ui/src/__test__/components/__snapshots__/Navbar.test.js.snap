// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Can see navbar 1`] = `
<div
  className="ui tiny borderless fluid menu"
>
  
  <a
    className="header item"
    onClick={[Function]}
  >
    NextLens.io
  </a>
  <div
    className="right menu"
  >
    <div
      className="item"
      onClick={[Function]}
    >
      
    </div>
    <div
      className="item"
      onClick={[Function]}
    >
      <button
        className="ui tiny primary button"
        onClick={[Function]}
      >
        Log in
      </button>
    </div>
  </div>
</div>
`;
